src/ccl/charts/chart.component.html

```diff
@@ -1,13 +1,50 @@
 <div class="chart-heading">
-  <h2 class="chart-label">
+    <h2 class="chart-label">
     {{indicator}}
-  </h2>
-  <div class="chart-actions">
-    <button class="button button-link" data-toggle="tooltip" data-placement="bottom" data-delay="300" title="" data-original-title="Export"><i class="icon icon-export"></i></button>
-    <button class="button button-link" data-toggle="tooltip" data-placement="bottom" data-delay="300" title="" data-original-title="Options"><i class="icon icon-cog"></i></button>
-  </div>
+    </h2>
+    <div class="chart-actions">
+        <button class="button button-link" data-toggle="tooltip" data-placement="bottom" data-delay="300" title="" data-original-title="Export"><i class="icon icon-export"></i></button>
+        <button class="button button-link" data-toggle="tooltip" data-placement="bottom" data-delay="300" title="" data-original-title="Options"><i class="icon icon-cog"></i></button>
+    </div>
 </div>
 <div class="chart-body">
-  <line-graph [data]="chartData" [indicator]="indicator" [trendline]="trendline"></line-graph>
-  <button (click)="toggleTrendline()">Trendline</button>
+    <line-graph
+        [data]="chartData"
+        [indicator]="indicator"
+        [trendline]="trendline"
+        [min]="min"
+        [max]="max"
+        [minVal]="minVal"
+        [maxVal]="maxVal">
+    </line-graph>
 </div>
+<div class="chart-controls">
+    <h5>Options</h5>
+    <ui-switch (change)="toggleTrendline()" size="small"></ui-switch>Show Trendline
+    <ui-switch (change)="toggleMin()" size="small"></ui-switch>Show Min
+    <form>
```

Comment:
I tried using inputs (without ngForms) first but it didn't work as planned. IDR why though, but i can give it another go and find out